docs/threadpools.md

```diff
@@ -0,0 +1,33 @@
+Starlette supports both `async` and `sync` routes and background tasks.
+The sync part is running inside a threadpool by using [`AnyIO`](https://github.com/agronholm/anyio).
+
+The `AnyIO` library by default starts a threadpool with a fixed number of threads, which works for most cases.
+But since the threadpool is shared by both incoming requests, background tasks, and any other AnyIO thread pool usage it might be useful to control the number of threads.
+
+
+```py
+import anyio
+from starlette.applications import Starlette
+from starlette.background import BackgroundTasks
+from starlette.request import Request
+from starlette.responses import JSONResponse
+from starlette.routing import Route
+
+# Set number of threads in threadpool to 100
+anyio.to_thread.current_default_thread_limiter().total_tokens = 100
```

Comment:
Doesn't this needs to be in the lifespan if using multiple workers?

Reply:
I think each worker will do this, so there’s no difference.

but I can do a quick test to confirm